firefox_ui_tests/functional/locationbar/test_star_in_autocomplete.py

```diff
@@ -2,7 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
Can we keep the former name of the test please? `test_suggest_bookmarks.py` is more descriptive here and does not tell how Firefox is showing bookmarks. It might change in the future.